src/trade-params/TradeParams.js

```diff
@@ -182,6 +196,11 @@ export default class TradeParams extends Component {
         this.updateTradeParams(updatedAmountPerPoint);
     }
 
+    onCloseModal = () => {
+        const { actions, index } = this.props;
```

Comment:
I think it's a good idea, just we need to do it in a way that UI does not breakdown in mobile